.github/ISSUE_TEMPLATE/airflow_bug_report.yml

```diff
@@ -25,7 +25,7 @@ body:
         the latest release or main to see if the issue is fixed before reporting it.
       multiple: false
       options:
-        - "2.10.0"
+        - "2.10.1"
         - "2.9.3"
```

Comment:
We should remove 2.9.3

Reply:
Hey @ephraimbuddy, I have removed it. PTAL.